@Atharv3221 , please fix.
```
                       ^
src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodMalformedJavadoc.java:26: error: unexpected content
     * @see {@link org.infinispan.cache.impl.CacheSupport#set(Object, Object)}
                                                                             ^
src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodMalformedJavadoc.java:47: error: malformed HTML
     * @author Sanne Grinovero <abc@infinispan.org> (C) 2011 Red Hat Inc.
```

--- a/src/it/java/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/WhereJavadocIsUsedTest.java
+++ b/src/it/java/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/WhereJavadocIsUsedTest.java
@@ -55,4 +55,9 @@ public class WhereJavadocIsUsedTest extends AbstractGoogleModuleTestSupport {
         verifyWithWholeConfig(getPath("InputJavadocTypeOnRecord2.java"));
     }
 
+    @Test
+    public void testJavadocMethodProtected() throws Exception {
+        verifyWithWholeConfig(getPath("InputJavadocMethodProtected.java"));
+    }
+
 }

--- a/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/InputJavadocMethodProtected.java
+++ b/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/InputJavadocMethodProtected.java
@@ -1,0 +1,38 @@
+package com.google.checkstyle.test.chapter7javadoc.rule73wherejavadocrequired;
+
+/** Some javadoc. */
+public class InputJavadocMethodProtected {
+
+  /**
+   * Some javadoc.
+   *
+   * @param b this parameter doesn't exist
+   */
+  protected void foo(int a) {}
+
+  // violation 4 lines above 'Unused @param tag for 'b''
+
+  /**
+   * Some javadoc.
+   *
+   * @param a this parameter do exist
+   */
+  protected void foo1(int a) {}
+
+  /**
+   * Some javadoc.
+   *
+   * @param b parameter exist
+   * @param c paramter doesn't exist
+   */
+  protected void foo2(int b) {}
+
+  // violation 4 lines above 'Unused @param tag for 'c''
+
+  /**
+   * Some javadoc.
+   *
+   * @param b parameter exist
+   */
+  protected void foo3(int b) {}
+}

--- a/src/main/resources/google_checks.xml
+++ b/src/main/resources/google_checks.xml
@@ -438,7 +438,7 @@
                                   VARIABLE_DEF, RECORD_DEF, COMPACT_CTOR_DEF"/>
     </module>
     <module name="JavadocMethod">
-      <property name="accessModifiers" value="public"/>
+      <property name="accessModifiers" value="public, protected"/>
       <property name="allowMissingParamTags" value="true"/>
       <property name="allowMissingReturnTag" value="true"/>
       <property name="allowedAnnotations" value="Override, Test"/>

